### PR TITLE
Community

### PR DESCRIPTION
**Pull request by jhabdas**
Add [community article](https://habd.as/pagespeed-100-with-jekyll-s3-and-cloudfront/) I wrote on using this Gem, hosted on [a website](github.com/jhabdas/habd.as) built using this Gem. Also fixes some linkrot in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ and [Java](http://java.com) to run. (S3_website is partly written in Scala, henc
 Here's how you can get started:
 
 * Create API credentials that have sufficient permissions to S3. More info
-  [here](additional-docs/setting-up-aws-credentials.md).
+  [here](//github.com/laurilehmijoki/s3_website/tree/master/additional-docs/setting-up-aws-credentials.md).
```
Comment (pathawks) [inline README.md]: Should probably use `https://` explicitly instead of `//`

Comment (jhabdas) [inline README.md]: All set

**Comment by jhabdas** [comment]
Added another blog post

**Comment by jhabdas** [comment]
Thank you kindly!

**Comment by laurilehmijoki** [comment]
Thanks for the PR!

**Pull request merged**